### PR TITLE
Update dependencies to latest version

### PR DESCRIPTION
**Pull request by FTheron**
I upgraded the dependencies to their latest versions.

I also needed to make some [changes](https://nextjs.org/docs/messages/invalid-new-link-with-extra-anchor) for next.js 13.

### CODE REVIEW

File: src/navigation/NavbarTwoColumns.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ type INavbarProps = {
 const NavbarTwoColumns = (props: INavbarProps) => (
   <div className="flex flex-wrap justify-between items-center">
     <div>
-      <Link href="/">
+      <Link href="/" legacyBehavior>
```
Comment (ixartz) [inline src/navigation/NavbarTwoColumns.tsx]: The PR looks good to me, just one question about the `legacyBehavior`. it's not possible to use the modern `<Link>` component?

**Comment by ymmtyuhei** [comment]
Thank you for creating this template. Very useful and helpful.

I am having problems for React 18 is not available. I hope this issue will be resolved.

**Comment by ixartz** [comment]
I was expecting a response from @FTheron about one change

Without any response, I'll work on it this weekend

**Comment by FTheron** [comment]
I somehow missed the first notification. I'll see if I can update that to use the modern link component tonight. 👍

**Comment by FTheron** [comment]
I fixed the navbar link and updated the dependencies to be up to date as of today. Did a quick check and everything looks good.

**Comment by ixartz** [comment]
@FTheron Thank you for taking the take to update the PR and your help!

**Pull request merged**